lib/endpoints/class-wp-rest-posts-terms-controller.php

```diff
@@ -0,0 +1,229 @@
+<?php
+
+class WP_REST_Posts_Terms_Controller extends WP_REST_Controller {
+
+	protected $post_type;
+
+	public function __construct( $post_type ) {
```

Comment:
Need to pass `taxonomy` here too, as the route should be explicitly defined based on the taxonomy. See #1170